spark_auto_mapper/automappers/automapper.py

```diff
@@ -19,6 +19,8 @@
 
 TEMPORARY_KEY = "__row_id"
 
+raise KeyError("WTF!?!?!?!?!")
```

Comment:
what does this do?

Reply:
Whoops. That was supposed to be removed. A debug issue. Sorry.

Reply:
Removed.